packages/config/src/index.ts

```diff
@@ -104,20 +105,20 @@ export default class ConfigManager implements IConfigManager {
           }
         });
       })
-      .catch((err) => {
-        console.log('Failed to recover state');
+      .catch((_) => {
```

Comment:
When variable is not in use, you remove it. Don't add "_", they have absolutely no use.